src/extent.c

```diff
@@ -1105,6 +1126,13 @@ extent_grow_retained(tsdn_t *tsdn, arena_t *arena,
 		goto label_err;
 	}
 
+	/*
+	 * There are situations in which the Linux kernel will include
```

Comment:
How do you feel about:

When core-dumping, the Linux kernel will include the whole address space in its core dumps, even pages we haven't touched yet. This can increase the size of core dumps, and the amount of time it takes to create them. To avoid these costs, we'll set the memory as undumpable until we're ready to use it.